src/annotator/anchoring/test/xpath-evaluate-test.js

```diff
@@ -1,8 +1,8 @@
 import $ from 'jquery';
 
-import { nodeFromXPath, xpathFromNode, $imports } from '../range-js';
+import { nodeFromXPath, xpathFromNode, $imports } from '../xpath-evaluate';
 
-describe('annotator/anchoring/range-js', () => {
+describe('annotator/anchoring/xpath-evaluate', () => {
```

Comment:
`xpath-evaluate` contains functions for both evaluating and generating xpaths. I would be inclined to name it just `xpath.js`.